src/Uploader/useUploader.ts

```diff
@@ -68,6 +68,9 @@ export default function useUploader({
     onAdd && onAdd(newSelection);
   };
 
+  // get the accepted file type extensions
+  const acceptedFileExtensions = Object.values(acceptedFiles ?? {}).flat();
```

Comment:
please apply a .toLowerCase() to avoid case sensitivity. E.g. right now, ".PNG" is not accepted in the image uploader of FLEET because of this